pkg/rc/farm.go

```diff
@@ -65,22 +67,27 @@ func NewFarm(
 	logger logging.Logger,
 	rcSelector klabels.Selector,
 	alerter alerting.Alerter,
+	errorReporter error_reporter.Reporter,
```

Comment:
i did do that for metrics registry in the farms I think, but i also don't mind the `nil`